packages/common/src/types/user.ts

```diff
@@ -32,3 +32,16 @@ export interface UserProperty {
 export interface FakeLogin {
   hsaId: string
 }
+
+export interface CertificateType {
```

Comment:
Maybe this belongs somewhere else since it's not directly connected to the user.

Reply:
I moved it to patient since the certificate types belongs to the patient. I also moved the state of certificate types to patient.